lib/puppet/provider/cisco_bgp/cisco.rb

```diff
@@ -209,6 +209,28 @@ def timer_bestpath_limit_set
     @bgp_vrf.timer_bestpath_limit_set(limit, always)
   end
 
+  def image_older_version
+    fd = Facter.value('cisco')
+    image = fd['images']['system_image']
+    pid = fd['inventory']['chassis']['pid']
+    image[/7.0.3.I2|I3|I4/] || pid[/N(5|6|7|8)/]
```

Comment:
Is there a reason we are not using the image version for N5|6|7|8k platforms?  If possible let's use the image version so that if/when images on these platforms support the new cli we don't have to make a change in the code base to enable the new cli.